src/pages/CoursesListView/index.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <div>
+    <div class="home-header">
+      <h2>Courses</h2>
+
+      <q-select
+        class="input"
+        v-model="sortingOption"
+        dark
+        :options="sortingOptions"
+        label="Sort by"
+      />
+    </div>
+
+    <div class="home-coursesList">
+      <q-inner-loading
+        :showing="!useGlobalStore().coursesLists.length"
+        dark
+        label="Please wait..."
+        label-class="text-teal"
+        label-style="font-size: 1.1em"
+      />
+      <Course
+        v-for="{
+          id,
+          title,
+          rating,
+          lessonsCount,
+          meta,
+          description,
+          previewImageLink
+        } in useGlobalStore().paginatedCoursesLists"
+        :key="id"
+        :title="title"
+        :rating="rating"
+        :lessonsCount="lessonsCount"
+        :skills="meta['skills']"
+        :description="description"
+        :previewImageLink="previewImageLink"
+        :courseVideoPreview="meta['courseVideoPreview']"
+        :id="id"
+        class="home-course"
+      />
+    </div>
+
+    <q-pagination
+      v-model="pagination"
+      @update:model-value="onPagination"
+      color="yellow"
+      active-text-color="dark"
+      :max="maxPages"
+      :max-pages="maxElementsOnPage"
+      boundary-numbers
+      direction-links
+      class="home-pagination"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, onMounted, computed } from 'vue'
+import Course from './Course.vue'
+import { getPreviewCourses } from '@/api'
+import { useGlobalStore } from '@/stores/global'
+
+onMounted(async () => {
+  const coursesLists = await getPreviewCourses()
+  useGlobalStore().setCoursesLists(coursesLists.data.courses)
```

Comment:
I think you fixed the bug when you added the .env.development file, the problem was that without the .env the api didn't work, so the error was there. 

It would be nice if you could add the .env point to the ReadMe